src/uk/gov/hmcts/cmc/integrationtests/IntegrationTests.groovy

```diff
@@ -32,22 +32,22 @@ class IntegrationTests implements Serializable {
   }
 
   def execute(Map config, Team team = Team.CITIZEN) {
```

Comment:
Do we still need `Team` class? If we do we can at least remove it from this method signature as it is no longer in use.

Reply:
We don't. I wanted to remove it step by step:
1. merge all new code in,
2. update all pipelines to use the new code,
3. remove `Team` after we're sure it's not used anywhere anymore.